WalletWasabi/Stores/IndexStore.cs

```diff
@@ -39,9 +39,14 @@ public IndexStore(string workFolderPath, Network network, SmartHeaderChain smart
 			File.Delete(NewIndexFilePath);
 		}
 
+		IndexStorage = CreateIndexStorage();
+	}
+
+	private BlockFilterSqliteStorage CreateIndexStorage()
```

Comment:
I called it that way since we set IndexStorage with it (it doesn't set directly to make it easier to use in the constructor).
If others are vote for it, I commit the suggestion, as I am not familiar with the project's naming conventions yet.